app/helpers/curate_helper.rb

```diff
@@ -182,11 +177,24 @@ def sorted_college_list_for_generic_works
       list[k]["label"]
     end.sort << "Other"
   end
-
+    def sorted_college_list_for_degrees
```

Comment:
Curious why you left this here and added `sorted_college_list_for_degrees_new`. If it's not being used anywhere else,  it would be better to just change this method in place instead of adding the new one.